src/piece.cpp

```diff
@@ -110,6 +110,14 @@ namespace {
       p->stepsCapture = {2 * SOUTH_WEST, 2 * SOUTH_EAST, 2 * NORTH_WEST, 2 * NORTH_EAST};
       return p;
   }
+  PieceInfo* dabbaba_piece() {
+      PieceInfo* p = new PieceInfo();
+      p->name = "dabbaba";
+      p->betza = "D";
```

Comment:
I'm guessing the Betza notation isn't important for the use case I'm showing (xboard via terminal), which is indeed all I need -- added this anyway, but not sure if "D" needs to be defined elsewhere in Fairy-Stockfish.